### ISSUE TITLE
Compatibility with ollama

### ISSUE THREAD

**Issue by igolaizola** [hitz-zentroa/latxa#1]
First of all, thank you for this awesome project!
I heard about it this morning on the radio and I couldn't resist getting my hands on it.

I was trying to import your model into [ollama](https://github.com/ollama/ollama) but I haven't been successful in doing so.

I tried following this guide: https://github.com/ollama/ollama/blob/main/docs/import.md#importing-pytorch--safetensors, which attempts to convert the HuggingFace model and then quantize it. Even though I could complete all the steps without errors, the final model was producing incoherent responses.

Here is an example:
```
❯ ollama run latxa
>>> Kaixo latxa!
"}
{"text": "Irailaren 24an Zumaian hasiko da aurtengo zirkuitua\n\nBerrikuntzekin dator Euskal Herriko
Surf Federazioak antolatzen duen surf zirkuitoa. Alde batetik, datak aldatu dira. Orain arte urte amaieran
egiten baitziren probak...
```

I have tried various quantifications without success. Probably, the conversion script suggested by this guide is not the correct one for your model.

The HuggingFace model I was using is this one: https://huggingface.co/HiTZ/latxa-7b-v1.1

Do you have any ideas on how I could make it compatible with ollama?

Thanks again!

**Comment by osainz59** [comment]
Hi @igolaizola !

We are aware of this issue and we are working on it. 

Thank you for your feedback :)

Oscar

**Comment by igolaizola** [comment]
Hi @osainz59! I'm glad you're working on it.
I was able to somehow make it work with a specific instruct version of the model: https://huggingface.co/xezpeleta/latxa-7b-instruct. However, it isn't the latest version as far as I know.

**Comment by xezpeleta** [comment]
> Hi @osainz59! I'm glad you're working on it. I was able to somehow make it work with a specific instruct version of the model: https://huggingface.co/xezpeleta/latxa-7b-instruct. However, it isn't the latest version as far as I know.

Hi @igolaizola , that's my fine-tuned version (not official), which is not very good as you can see. It was just an experiment :man_scientist: 

I hope HiTZ will soon release an instruct version of Latxa :crossed_fingers:

**Comment by igolaizola** [comment]
> > Hi @osainz59! I'm glad you're working on it. I was able to somehow make it work with a specific instruct version of the model: https://huggingface.co/xezpeleta/latxa-7b-instruct. However, it isn't the latest version as far as I know.
> 
> Hi @igolaizola , that's my fine-tuned version (not official), which is not very good as you can see. It was just an experiment 👨‍🔬

Thanks for the experiment anyway 😀

> I hope HiTZ will soon release an instruct version of Latxa 🤞

Looking forward it!

**Comment by juletx** [comment]
Hi @xezpeleta @igolaizola!

The JSON bug should be fixed in the latest Latxa v1.2 models: https://huggingface.co/HiTZ/latxa-7b-v1.2

We are working on new base and instruct models, but we do not know when they will be ready to release.
